### DISCUSSION TITLE
industry wont accept goods delivered by conveyor

### DISCUSSION THREAD

**Discussion by Skathmaine** [voxeltycoon/issues#641] [General] [answered]
i have tried conveying to industry, to a ware house connected to a station and to a station.  i have partial success in going to the warehouse connected to the station  but then the station says it accepts iron ore and has iron ore but it wont move the iron ore to the industry

**Comment by katepolak** [comment, marked as answer]
This is the intended behaviour, the only way businesses accept goods is when delivered by a vehicle. Take a look at #475 for discussion about possible changes to this in the future.
